examples/js/loaders/deprecated/LegacyGLTFLoader.js

```diff
@@ -3,6 +3,7 @@
  * @author mrdoob / http://mrdoob.com/
  * @author Tony Parisi / http://www.tonyparisi.com/
  * @author Takahiro / https://github.com/takahirox
+ * @author 1d2d3d / https://github.com/1d2d3d
```

Comment:
FYI - All contributions are welcome, but we typically do not add authorship for minor changes.

Reply:
We should change `@author` for `@maintainer`. I bet people would stop adding themselves though 😂